.github/actions/report-flaky/report_flaky.js

```diff
@@ -0,0 +1,26 @@
+// Taken from: https://nodejs.org/api/readline.html#readline_example_read_file_stream_line_by_line
+const fs = require('node:fs');
+const readline = require('node:readline');
+
+const core = require('@actions/core');
+
+const logFilePath = core.getInput('logFile');
+
+async function emitAnnotations() {
+  const fileStream = fs.createReadStream(logFilePath);
+
+  const rl = readline.createInterface({
+    input: fileStream,
+    crlfDelay: Infinity,
+  });
+  // Note: we use the crlfDelay option to recognize all instances of CR LF
+  // ('\r\n') in input.txt as a single line break.
```

Comment:
LOL. And pray you never encounter MacOS < X files... Those use only \r.

Reply:
copy and pasted from the nodejs docs and we run github actions on ubuntu-latest :feelsDabMan: